cmake/FindFFTW3.cmake

```diff
@@ -36,7 +38,9 @@ FIND_PACKAGE_HANDLE_STANDARD_ARGS(FFTW3_LONG   DEFAULT_MSG FFTW3_INCLUDE_DIR FFT
 # Adding threaded version of FFTW3
 if (FFTW3_THREADS_LIBRARIES)
   set (FFTW3_LIBRARIES ${FFTW3_LIBRARIES} ${FFTW3_THREADS_LIBRARIES})
-  set (FFTW3_DEP_LIBRARIES ${FFTW3_DEP_LIBRARIES} pthread)
+  if (NOT MSVC)
+    set (FFTW3_DEP_LIBRARIES ${FFTW3_DEP_LIBRARIES} pthread)
+  endif (NOT MSVC)
```

Comment:
Is it because pthread is not available on MSW?

Reply:
If we want to use pthread on Visual Studio, we have to install another dependance, while the compilation works without this library (pthread), including FFTW3 in cmake process.

Reply:
https://stackoverflow.com/questions/48894212/visual-studio-2017-how-to-make-include-pthread-h-work

Reply:
👌